upstairs/src/lib.rs

```diff
@@ -3608,7 +3937,9 @@ impl Guest {
      */
     pub fn show_work(&self) -> Result<WQCounts, CrucibleError> {
         if !self.is_active() {
-            return Err(CrucibleError::UpstairsInactive);
+            println!("Request for work from inactive upstairs");
+            // XXX Test access is allowed for now, but not forever.
```

Comment:
When is this needed?

Reply:
When I'm testing two clients at the same time.  The non-active client can still show some status.  Before todays storage meeting I was thinking we would support failover/failback situations, and this was part of that.

This will probably all go away at some point, the guest will not have access to the command that calls this, so we won't need to check for is_active.

Reply:
Something for a future PR though.